packages/studioCMS/src/integrations/studioCMSDashboard/routes/dashboard/components/ThemeManager.astro

```diff
@@ -27,8 +27,8 @@ const { defaultTheme = "auto" } = Astro.props;
     function applyTheme(theme) {
       document.documentElement.dataset.theme =
         theme === "auto" ? systemTheme : theme;
-      document.documentElement.dataset.theme === "dark" && document.documentElement.classList.add('sl-theme-dark');
-      document.documentElement.dataset.theme === "light" && document.documentElement.classList.remove('sl-theme-dark');
+        document.documentElement.dataset.theme === "dark" && document.documentElement.classList.add('sl-theme-dark');
+        document.documentElement.dataset.theme === "light" && document.documentElement.classList.remove('sl-theme-dark');
```

Comment:
????????????????????????????

Reply:
I think the file got linted.... i never touched this one